docker/environment.yml

```diff
@@ -64,17 +68,17 @@ dependencies:
   - xarray
   # xesmf: Universal Regridder for Geospatial Data
   - xesmf >= 0.8.9
+  # zarr: Python implementation of Zarr, a format for chunked, compressed, N-dimensional arrays
+  - zarr
```

Comment:
zarr, h5netcdf, flox are dependencies of xscen or xclim or ravenpy, we do not want them specified directly here to allow xscen/xclim/ravenpy to manage their version pinning if needed.

Reply:
The versions are managed there. These entries are here to ensure that that we pull the right versions for the image. Agree that leaving them unpinned should be the default. The pins currently here are the bare minimum supported versions.

Reply:
By experience, by simply mentionning a dependency here, is like asking for the latest version of that package.  And since it is mentionned here, it has higher precedence than the pin in other dependencies.  If you want xscen or xclim or ravenpy to manage those version, better do not mention them directly here.